andela/archangel-backend
# #167730616 Configure PostgreSQL as Database Engine

#### What does this PR do?
Have PostgreSQL as the database engine for Barefoot Nomad

#### Description of Task to be completed?
Optimize PostgreSQL database engine for production-ready environments

#### How should this be manually tested?
After cloning the repo, open a package.json file in a code editor precisely VSCode and check for a block of code with an object name of `dependencies` to find two names `(pg and pg-hstore)`. Then check on `env.sample` to connect to the database

#### Any background context you want to provide?
- Adding postgresql configuration file
• Creating a pool
• Successfully creating the database [barefoot]
• Creating all the tables to be used in the models' folder
• Configuring the application to start the server with Sequelize properly integrated
• Bootstrapping the database folder by creating auto-generated boiler-plates
• Updating script files for easier run and update
• Creating the seeders' files to insert datas into the tables which could be ran with the script (npm run create:tables)

#### What are the relevant pivotal tracker stories?
[#167730616](https://www.pivotaltracker.com/story/show/167730616)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/28824477/63648188-db631f80-c723-11e9-8461-7370400a3e4e.png)




#### Questions:
